libyui-ncurses-pkg/CMakeLists.txt

```diff
@@ -30,8 +30,25 @@ option( WERROR            "Treat all compiler warnings as errors"     on  )
 #----------------------------------------------------------------------
 
 
-# As of now, no support for /usr/local instead of /usr. But CMake DESTDIR works.
-set( CMAKE_INSTALL_PREFIX /usr )
+# We use /usr as the default CMAKE_INSTALL_PREFIX, but it can be set on the
```

Comment:
Having the very same code in all the *CMakeLists.txt* files is too repetitive. Maybe we can move all this common code to a *config.cmake* file at the top level directory and then include it whenever it is needed.

Reply:
This is the one thing I want to avoid. That makes the files no longer self-contained.

The repetetive stuff is the result of change requests upon change requests and trying to please everyone.